webdriver/tests/close_window/user_prompts.py

```diff
@@ -3,7 +3,7 @@
 from tests.support.asserts import assert_error, assert_dialog_handled
 from tests.support.fixtures import create_dialog, create_window
 from tests.support.inline import inline
-
+from time import sleep
```

Comment:
Please remove the line which you probably used when debugging the code.

Reply:
Yep. Will do.